OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -354,18 +354,21 @@ static class DMOpcodeHandlers {
                 Dictionary<DreamValue, DreamValue> argListNamedValues = argList.GetAssociativeValues();
 
                 foreach (DreamValue value in argListValues) {
-                    if (!argListNamedValues.ContainsKey(value)) {
+                    if (argListNamedValues != null && !argListNamedValues.ContainsKey(value)) {
```

Comment:
I feel like it may be better to make `DreamList` have `ContainsKey` helpers and similar so that usage code doesn't have to keep in mind that the dict can be null. This sounds like it'd be very error prone otherwise.

Also applies to other changes below.

Reply:
Agreed